_posts/2025-01-09-v5-1-latest-release.md

```diff
@@ -0,0 +1,203 @@
+[proposed LTS strategy ](https://github.com/expressjs/discussions/pull/352) which will be the basis for our forthcoming
+docs. Input is very welcome on this so we can make sure it is clearly communicated and acceptable to the community.
+
+Additionally since then we have announced a partnership with HeroDevs to help companies who are less capable of
+updating. More information on how this will work when v4 EOL will come when we get closer to that time.
+
+---
+
+## Support Phases and Going latest
+
+What does it mean to "go latest"? If you are unfamiliar with how npm `dist-tags` work, the `latest` tag is what users
+will get when they run `npm install express`. This is important because it means it is the "default installed version"
+and will trigger the transition of nearly 17 million weekly downloads from our current latest v4.21.2 to v5. As we start
+this transition we want users, companies, and other organizations to know exactly what it means for support. To help
+with this we have developed an LTS strategy which defines our 3 support phases and set's target dates for when v4 will
+enter EOL.
+
+Express major versions will go through three supported phases:
+
+  - `CURRENT`: A new major version is designated as `CURRENT` upon release. It is available but not the `latest` version
+    on npm for a minimum of 3 months.
+  - `ACTIVE`: After the minimum 3 month period and the TC agrees it is stable and secure, the `ACTIVE` version is
+    tagged `latest` on npm for a minimum of 12 months.
+  - `MAINTENANCE`: When a new major version becomes `ACTIVE`, the previous major version enters `MAINTENANCE` for 12 months.
+
+### CURRENT
+
+- New majors will go through a short period of hardening to ensure stability, security, and ecosystem libraries/resources
+  compatibility.
+- We will strive to ensure no breaking changes are included, but reserve the right to make security or high priority
+  fixes of breaking nature within this period.
+- `CURRENT` lines will receive all types of active work including: bug fixes, security patches, new features, and
+  deprecation notices.
+- Users are recommended to use  `CURRENT` lines and to upgrade as quickly as their risk profile allows
+
+### ACTIVE
+
+- `ACTIVE` lines will receive all types of active work including: bug fixes, security patches, new features, and
+  deprecation notices.
+- For users, `ACTIVE` lines are considered the most stable and well supported version at any given time.
+
+### MAINTENANCE
+
+- `MAINTENANCE` lines will only receive security patches or high priority bug fixes.
+- Users are highly encouraged to upgrade to a `CURRENT` or `ACTIVE` release.
+
+### Proposed Schedule
+
+For the existing release lines, we will set the following phase dates:
+
+
+| Major | CURRENT | ACTIVE | MAINTENANCE | EOL |
+| ----- | ------- | ------ | ----------- | --- |
+| 4.x   |         |        | 2025-04-01 | 2026-10-01 or later |
```

Comment:
```suggestion
| 4.x   |         |        | 2025-04-01 | 2026-10-01 |
```

Just cross-posting: https://github.com/expressjs/discussions/pull/352#discussion_r2019981355

Reply:
I can update this once we resolve the conversation in the other thread, but: https://github.com/expressjs/discussions/pull/352#discussion_r2020173430